packages/framework/client-logger/app-insights-logger/src/fluidAppInsightsLogger.ts

```diff
@@ -39,15 +39,29 @@ export interface FluidAppInsightsLoggerConfig {
 
 /**
  * Object used with an {@link FluidAppInsightsLoggerConfig}
- * to define logic for filtering of telemetry events
+ * to define a filter with logic for matching it to telemetry events.
+ * Filters can include either a category, namespace or both types of filter; A valid filter must have at least one defined.
  *
  * @public
  */
 export interface TelemetryFilter {
 	/**
 	 * The category of telemetry event that this filter applies to
 	 */
-	category: TelemetryEventCategory;
+	category?: TelemetryEventCategory;
+	/**
+	 * The namespace pattern to filter telemetry events.
+	 * For example, "perf:latency:*" will match any namespace starting with "perf:latency:"
+	 * This is a regex pattern.
```

Comment:
The example you gave was in terms of glob-style patterns ("perf:latency:*"), rather than as a RegEx. Is this correct?

Reply:
I'm not entirely sure what the difference is here, if you want to explain but also we moved to `.startsWith()` so this may no longer be a relevant discussion

Reply:
It's just that the example RegEx didn't handle the case being described. `perf:latency:*` would match the literal string `perf:latency` followed by 0 or more instances of `:`. The desired RegEx would have been `perf:latency:.*` (or `perf:latency:.+` if you wanted to ensure at least one more character after the `:`).

But moving to `startsWith` resolves it :)